### PR TITLE
Initial portfolio

### PR DESCRIPTION
**Pull request by xiaoalem**
SPS-week1 web-development initial version

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,18 +2,94 @@
...
+
+      <!-----------Project---------->
+      <section class="project" id="project">
+          <div class="heading">
+              <h2>Projects</h2>
+          </div>
+          <div class="project-content">
+            <div class="col">
+                <div class="col-img">
+                  <img src="images/winter.jpg" style="width: auto; height: auto;max-width: 540px;max-height: 540px">
```
Comment (sbahaud) [inline portfolio/src/main/webapp/index.html:75]: The image size seems to consistent across multiple images. Might be  a good idea to move to a css class.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,202 @@
...
-  margin-right: auto;
-  width: 650px;
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: 'Jost', sans-serif;
+    list-style: none;
+    text-decoration: smooth;
+}
```
Comment (sbahaud) [inline portfolio/src/main/webapp/style.css:8]: I recommend keeping the properties in a css class in alphabetical order. It makes it more readable. I won't strictly enforce it for SPS. But overall a good thing to follow,
box-sizing:
font-family:
list-style:
margin:
padding:
tetx-decoration:

**Review by sbahaud — APPROVED** [review]
Added a minor comment about ordering properties within a css. I don't strictly enforce that for SPS. so approving the PR.

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (xiaoalem) [reply portfolio/src/main/webapp/style.css:8]: That's a very good point!

**Pull request merged**